sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/AzureMonitorOptions.cs

```diff
@@ -36,6 +37,11 @@ public class AzureMonitorOptions : ClientOptions
         /// </summary>
         public bool DisableOfflineStorage { get; set; }
 
+        /// <summary>
+        /// Enables or disables the Live Metrics feature.
```

Comment:
link to public doc.
Given this is enabled by default - would be nice to clarify right here that
1. this incur no additional billing/cost.
2. There is perf overhead due to additional data collection/processing/networking calls, but that is paid *only* when someone opens LiveMetrics portal in the UI, and when the portal is closed, LiveMetrics go back to "silent" mode with near zero overhead.